README.md

```diff
@@ -139,7 +139,29 @@ To run kaniko in Docker, run the following command:
 
 kaniko uses Docker credential helpers to push images to a registry.
 
-kaniko comes with support for GCR and Amazon ECR, but configuring another credential helper should allow pushing to a different registry.
+kaniko comes with support for GCR, Docker `config.json` and Amazon ECR, but configuring another credential helper should allow pushing to a different registry.
+
+#### Pushing to Docker Hub
+
+Get your docker registry user and password encoded in base64
+
+    echo USER:PASSWORD | base64
+
+Create a `config.json` file with your Docker registry url and the previous generated base64 string
+
+```
+{
+	"auths": {
+		"https://index.docker.io/v1/": {
+			"auth": "xxxxxxxxxxxxxxx"
+		}
+	}
+}
+```
+
+Run kaniko with the `config.json` inside `/root/.docker/config.json`
```

Comment:
This should now be `/kaniko/.docker/config.json` because of https://github.com/GoogleContainerTools/kaniko/commit/e3f4dc479d03075d4eed7beb67be82c6e2e69f75

Reply:
Thanks for this comment @andrewrynhard . I had the same issue and pointing it to /kaniko/.docker/config.json solved NOT AUTHENTICATED issue.

Reply:
thanks!! done!!